docs/src/pages/docs/crocks/Writer.md

```diff
@@ -0,0 +1,384 @@
+---
+title: "Writer"
+description: "Writer Crock"
+layout: "guide"
+functions: ["log", "read"]
+weight: 160
+---
+
+
+```haskell
+Monoid m => Writer m a
+```
+
+The `Writer` monad makes it possible to represent and compose computations 
+that produce some additional data along with their computed values. `Writer` 
+is a `Product` type that consists of a `Monoid` as the first value that 
+typically holds metadata, while the second value holds the result of a 
+computation. 
+
+<article id="topic-implements">
+
+## Implements
+`Functor`, `Apply`, `Chain`, `Applicative`, `Monad`
+
+</article>
+
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Writer :: MonoidTypeRep M, Monoid m => M -> (m, a) -> Writer m a
+```
+
+`Writer` is a type constructor that takes a `Monoid` constructor and returns 
+a new constructor. The new constructor is a binary function that accepts a 
+`Monoid` instance and a value `a`, and returns a `Writer` instance.
+
```

Comment:
@evilsoft can comment on this but i believe there is an issue starting a line with `this`, not going to block this for that but just putting a note here to get teh feedback